packages/flutter_tools/lib/src/windows/visual_studio_validator.dart

```diff
@@ -47,6 +47,18 @@ class VisualStudioValidator extends DoctorValidator {
         messages.add(ValidationMessage(_userMessages.windows10SdkVersion(windows10SdkVersion)));
       }
 
+      if (_visualStudio.cmakePath != null) {
+        final String? cmakeVersion = _visualStudio.getCmakeVersion();
+        if (cmakeVersion != null) {
+          if (double.parse(cmakeVersion) >= 3.15) {
+            messages.add(ValidationMessage(_userMessages.visualStudioCmakeVersion(cmakeVersion)));
+          } else {
+            status = ValidationType.partial;
+            messages.add(ValidationMessage.error(_userMessages.cmakeTooOld('3.15')));
```

Comment:
This is not actionable information for a developer on Windows, because they didn't install CMake directly. What they need to know is that their version of VS 2019 is too old and they should run its updater, not the details of how we decided that it's too old.

Reply:
True